src/controllers/revision.ts

```diff
@@ -431,7 +428,7 @@ export const updateRevisionPublicationDate = async (req: Request, res: Response,
         }
 
         await RevisionRepository.updatePublishDate(revision, publishAt);
-        const updatedDataset = await DatasetRepository.getById(dataset.id);
+        const updatedDataset = await DatasetRepository.getById(dataset.id, {});
```

Comment:
A lot of the time the dataset returned to the frontend is only used to verify the action completed and then redirect the user back to the tasklist or overview.

We don't need to join any relations in these cases.